src/olympia/reviewers/tests/test_views.py

```diff
@@ -3269,7 +3269,8 @@ def test_extra_actions_admin_disable_enable(self):
         assert response.status_code == 200
         doc = pq(response.content)
         assert doc('#clear_auto_approval_delayed_until')  # Is still there.
-        assert doc('#clear_auto_approval_delayed_until_unlisted')  # Is still there.
+        # Is still there.
+        assert doc('#clear_auto_approval_delayed_until_unlisted')
```

Comment:
we don't need this change - have you got the line length set correctly?  (we have line-length set at 88 for black & ruff in pyproject.toml)